fastlane/lib/fastlane/actions/hockey.rb

```diff
@@ -171,12 +171,11 @@ def self.run(options)
         case response.status
         when 200...300
           url = response.body['public_url']
-          version_url = url + "/app_versions/" + response.body['version']
-          Actions.lane_context[SharedValues::HOCKEY_DOWNLOAD_LINK] = version_url
+          app_url = self.puts_download_url(response.body)
```

Comment:
omg, yes in a rush :/.
Actually the change works on my project :).
But i'm very worried of other people's